asserts/internal/grouping.go

```diff
@@ -188,16 +187,16 @@ func (gr *Groupings) bitsetSerialize(g *Grouping) string {
 	return base64.RawURLEncoding.EncodeToString(b.Bytes())
 }
 
-var errSerializedLabel = errors.New("invalid serialized grouping label")
+const errSerializedLabelFmt = "invalid serialized grouping label: %v"
 
 // Deserialize reconstructs a grouping out of the serialized label.
 func (gr *Groupings) Deserialize(label string) (*Grouping, error) {
 	b, err := base64.RawURLEncoding.DecodeString(label)
 	if err != nil {
-		return nil, errSerializedLabel
+		return nil, fmt.Errorf(errSerializedLabelFmt, err)
 	}
 	if len(b)%2 != 0 {
-		return nil, errSerializedLabel
+		return nil, fmt.Errorf(errSerializedLabelFmt, "not divisible in 16-bits words")
```

Comment:
s/in/by/ ?

Reply:
I checked with @degville, what is needed to fix this for this case is s/in/into/